OpenRA.Mods.Common/Scripting/Global/MapGlobal.cs

```diff
@@ -101,6 +101,12 @@ public WPos CenterOfCell(CPos cell)
 			return Context.World.Map.CenterOfCell(cell);
 		}
 
+		[Desc("Returns the type of the tile at the target cell.")]
+		public string TileType(CPos cell)
```

Comment:
This is actually returning the Terrain Type, not the Tile Type.  Can you please rename it?